crates/bevy_core_pipeline/src/prepass/node.rs

```diff
@@ -84,6 +84,16 @@ impl Node for PrepassNode {
                 },
             }));
         }
+        if let Some(view_velocities_texture) = &view_prepass_textures.velocity {
+            color_attachments.push(Some(RenderPassColorAttachment {
+                view: &view_velocities_texture.default_view,
+                resolve_target: None,
+                ops: Operations {
+                    load: LoadOp::Clear(Color::rgb_linear(1.0, 1.0, 0.0).into()),
```

Comment:
[AMD docs](https://gpuopen.com/performance/#clears) recommend clearing to 0,0,0 (or 1,1,1) for fast clears.

Reply:
Not sure if it matters since the blue channel is never written, but I've switched it to 1.0.